documentation/docs/scenarios/xsa-deploy/files/.npmrc

```diff
@@ -0,0 +1,11 @@
+# This file can be ommitted in the project if it is ensured
+# that the corresponding configuration is provided on a
+# higher level in the npm config (either on user level ~/.npmrc or
+# globally). For more details with regards to configuring npm check
+# man pages for npm-config/npmrc
+
+# The public npm registry from where to fetch e.g. Grunt
+registry=https://registry.npmjs.org
+
+# The SAP npm registry from where to fetch SAP specific Grunt modules
+@sap:registry=https://npm.sap.com
```

Comment:
lets use the new ui5 build (no grunt, so sap reg required)
https://sap.github.io/ui5-tooling/pages/CLI/#build

cf fiori best practice (WIP)

Reply:
What does that mean? We have to reference other dependencies in the node build? That means from the pipeline level in the narrower sense nothing changes, but dependencies in the `package.json`, content in `grunt.js` et al. has to be adjusted? --> If that is `true`: I guess we have to provide another example project.

Reply:
```json
"scripts": {
     "build": "ui5 build --a"
}, 
"devDependencies": {
    "@ui5/cli": "^1.2.0",
    ...
```

Pipeline exec remains the same. UI5 tooling can be retrieved from nodejs.

cf https://github.com/1084565/openui5-sample-app/blob/master/package.json

Reply:
npm file has been dropped in the meantime since we do not want to describe technology specific parts here (we are agnostic here to e.g. `node`, `java` when describing the deploymnet. I guess you comment in the light of that strategy your comment is void.

Reply:
Correct.